Formula/cyrus-sasl.rb

```diff
@@ -0,0 +1,39 @@
+class CyrusSasl < Formula
+  desc "Simple Authentication and Security Layer"
+  homepage "https://www.cyrusimap.org/sasl/"
+  url "https://github.com/cyrusimap/cyrus-sasl/releases/download/cyrus-sasl-2.1.27/cyrus-sasl-2.1.27.tar.gz"
+  sha256 "26866b1549b00ffd020f188a43c258017fa1c382b3ddadd8201536f72efb05d5"
+
+  keg_only :provided_by_macos
+
+  depends_on "openssl@1.1"
+
+  def install
+    system "./configure",
+      "--disable-macos-framework",
+      "--disable-dependency-tracking",
+      "--disable-silent-rules",
+      "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.cpp").write <<~EOS
+      #include <sasl/saslutil.h>
+      #include <assert.h>
+      #include <stdio.h>
+      int main(void) {
+        char buf[123] = "\\0";
+        unsigned len = 0;
+        int ret = sasl_encode64("Hello, world!", 13, buf, sizeof buf, &len);
+        assert(ret == SASL_OK);
+        printf("%u %s", len, buf);
+        return 0;
+      }
+    EOS
+
+    system ENV.cxx, "-o", "test", "test.cpp",
+      "-I#{Formula["cyrus-sasl"].include}", "-L#{Formula["cyrus-sasl"].lib}", "-lsasl2"
```

Comment:
```suggestion
      "-I#{include}", "-L#{lib}", "-lsasl2"
```

Reply:
Thanks! I thought that code looked weird.